pkgs/by-name/as/aseprite/package.nix

```diff
@@ -12,42 +12,60 @@
 let
   skia = callPackage ./skia.nix {};
 in
-stdenv.mkDerivation rec {
+clangStdenv.mkDerivation (finalAttrs: {
```

Comment:
Why to change `stdenv` to `clangStdenv`?

Reply:
Upstream recommends against compiling with gcc: https://github.com/aseprite/aseprite/blob/main/INSTALL.md#gcc-compiler
So I thought it's better to use clang by default